src/app/inventory/ItemIcon.m.scss

```diff
@@ -71,15 +71,45 @@ $commonBg: #366f42;
 }
 
 .animatedBackground {
-  display: none;
+  display: block;
+  transition: opacity 0.2s ease-out;
+  opacity: 0;
 
   @media (prefers-reduced-motion: no-preference) {
     :global(.item):hover & {
-      display: block;
+      opacity: 1;
     }
   }
 }
 
+.hoverFadeIn {
+  display: block;
+  transition: opacity 0.2s ease-out;
+  opacity: 0;
+
+  @media (prefers-reduced-motion: reduce) {
+    transition-duration: 0s;
+  }
```

Comment:
I might rather only apply the transition when this is `no-preference` rather than overriding it.